rolling-shutter/keyperimpl/gnosis/database/sql/queries/gnosiskeyper.sql

```diff
@@ -27,16 +34,9 @@ SET block_hash = $1, block_number = $2, slot = $3;
 -- name: GetTransactionSubmittedEventsSyncedUntil :one
 SELECT * FROM transaction_submitted_events_synced_until LIMIT 1;
 
--- name: SetTransactionSubmittedEventCount :exec
-INSERT INTO transaction_submitted_event_count (eon, event_count)
-VALUES ($1, $2)
-ON CONFLICT (eon) DO UPDATE
-SET event_count = $2;
-
 -- name: GetTransactionSubmittedEventCount :one
-SELECT event_count FROM transaction_submitted_event_count
-WHERE eon = $1
-LIMIT 1;
+SELECT max(index) + 1 FROM transaction_submitted_event
```

Comment:
This should be true with the reorg resistance but could probably be false beforehand